build/android/Makefile

```diff
@@ -198,7 +192,11 @@ $(PATHCFGFILE) :
 		echo "$$SDKFLDR is not a valid folder";                                \
 		exit 1;                                                                \
 	fi;                                                                        \
-	echo "SDKFOLDER = $$SDKFLDR" >> ${PATHCFGFILE};
+	echo "SDKFOLDER = $$SDKFLDR" >> ${PATHCFGFILE};                            \
```

Comment:
Why do this? It is pretty consistent inside the file to not add an \ at the end of the last line.